src/app/mycart/mycart.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-mycart',
+  templateUrl: './mycart.component.html',
+  styleUrls: ['./mycart.component.css']
+})
+export class MycartComponent implements OnInit {
```

Comment:
You should probably use CamelCase to name this as `MyCartComponent`, with `app-my-cart` as the selector